### DISCUSSION TITLE
can there be a way to contact a tech support of some kind?

### DISCUSSION THREAD

**Discussion by kitkatz2000** [community/community#27993] [Other Feature Feedback, Questions, & Ideas]
I have had a problem with one of my GitHub apps for almost a full week and have not been able to get in contact with a developer to help solve my problem. I was able to talk with someone who knows a lot a lot computers and has been kind enough to help me out with my issue the best they can, however we have had to create two different pages to try to get my issue fixed. If it is possible I would love to just be able to contact someone to help with my issue. thank you

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 

What type of support are you looking for? Our Support team is on hand for any GitHub-related issues and/or bugs, but they may not be who you're looking for if you need help with a specific project. 

You can find our Support team [here](https://support.github.com/contact).
